test/decompiler/vu_reference/jak2/ocean-vu0.txt

```diff
@@ -0,0 +1,132 @@
+    .word 0x40
+    .word 0x0
+    .word 0x20
+    .word 0x0
```

Comment:
I think these first 4 words are actually the header, and not part of the program.